lib/galaxy/files/templates/models.py

```diff
@@ -219,6 +219,24 @@ class eLabFTWFileSourceConfiguration(StrictModel):  # noqa
     writable: bool = True
 
 
+class InvenioFileSourceTemplateConfiguration(StrictModel):  # noqa
```

Comment:
```suggestion
class InvenioFileSourceTemplateConfiguration(StrictModel):
```

I used `# noqa` in #19493 because I wanted to violate the CamelCase convention

Reply:
Upps thanks, copy&paste leftover :sweat_smile: